### DISCUSSION TITLE
Allow to set prefered merge method

### DISCUSSION THREAD

**Discussion by rnestler** [community/community#13114] [Pull Requests]
Currently if more than one merge method is configured, the last merge method used or "Merge pull request" is selected by default in the UI:
<details><summary>Pull request settings</summary>

![screenshot-20220321-114820](https://user-images.githubusercontent.com/1435346/159246598-d6a7041b-21b7-4bea-8f28-8ca1b83f3f19.png)
</details>

![screenshot-20220321-114226](https://user-images.githubusercontent.com/1435346/159245703-c71f19f3-beea-4c79-a8fa-fe66bda4ac23.png)

I'd prefer to be able to define a default in the settings which is used instead of "Merge pull request". This would match our workflow of using Squash merge 90% of the time, but still allow to use "Merge pull request" if the developer crafted the individual commits carefully to provide more value.
Currently we only allow Squash merge, since otherwise developers would accidentally use the wrong merge method when they contribute to a new project the first time.

**Comment by SvenBunge** [comment]
Same here. It would also be a good idea to rethink the Settings UI. In Gitlab this Feature is possible and easy to configure.
